Runtime/CourseController/StandaloneCourseController.cs

```diff
@@ -529,7 +541,14 @@ private void PopulateChapterPickerOptions(int startingIndex)
                 chapterPicker.value = 0;
 
                 // If there is only one option, the dropdown is currently disabled.
-                chapterPicker.enabled = dropdownOptions.Count > 1;
+                if (dropdownOptions.Count > 1 == false)
```

Comment:
```suggestion
                if (dropdownOptions.Count < 2)
```
Seems more readable to me. But I am also fine without the change.